src/components/application_manager/src/commands/mobile/perform_audio_pass_thru_request.cc

```diff
@@ -62,7 +62,8 @@ void PerformAudioPassThruRequest::onTimeOut() {
   LOG4CXX_AUTO_TRACE(logger_);
 
   FinishTTSSpeak();
-  CommandRequestImpl::onTimeOut();
+  const std::string info = awaiting_ui_response_ ? "UI component does not respond":"";
```

Comment:
@pold500 I'm a bit confused, is it possible to get inside of this onTimeOut function having false "awaiting_ui_response_" parameter?